WalletWasabi/WabiSabi/Models/InputRegistrationRequest.cs

```diff
@@ -1,12 +1,13 @@
 using System;
-using System.Collections.Generic;
+using NBitcoin;
 using WalletWasabi.WabiSabi.Crypto.CredentialRequesting;
 
 namespace WalletWasabi.WabiSabi.Models
 {
-	public record InputsRegistrationRequest(
+	public record InputRegistrationRequest(
 		Guid RoundId,
-		IEnumerable<InputRoundSignaturePair> InputRoundSignaturePairs,
+		OutPoint Input,
+		byte[] RoundSignature,
```

Comment:
perhaps we should use `OwnershipProof` instead of `byte[]` here? it's the only type that's passed around as a serialized value in the high level code, and I'm not sure why

Reply:
Yes, this comes from the legacy code because it was a `compact signature` instead of a `ECDSASignature`.